src/Core/src/Platform/Android/JavaObjectExtensions.cs

```diff
@@ -1,17 +1,15 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Text;
 
-namespace Microsoft.Maui.Handlers
+namespace Microsoft.Maui
 {
-	public static class JavaObjectExtensions
+	static class JavaObjectExtensions
 	{
-		static bool IsDisposed(this Java.Lang.Object obj)
+		public static bool IsDisposed(this Java.Lang.Object obj)
```

Comment:
Can we also add the IsAlive variants?

https://github.com/dotnet/maui/blob/18e0f4ebbcca7c904ccdb67e6439cb72382e2a40/src/Compatibility/Core/src/Android/Extensions/JavaObjectExtensions.cs#L25

Reply:
It is there... lines 17+

Reply:
oh weird... Guess I needed to click the expanding thingie